styles.css

```diff
@@ -0,0 +1,92 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  width: 375px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  width: 375px;
```

Comment:
- [OPTIONAL] Kindly make sure your site is responsive by using % and vw as units of width to make your design fit all screen :recycle: